src/negbinfit.jl

```diff
@@ -139,7 +142,8 @@ function negbin(F,
         regmodel = glm(F, D, NegativeBinomial(θ), args...;
                        dropcollinear=dropcollinear, method=method, maxiter=maxiter,
                        atol=atol, rtol=rtol, verbose=verbose, kwargs...)
-        μ = regmodel.rr.mu
+            
+        μ = _rr(regmodel).mu
```

Comment:
Above the code seems to assume that `regmodel` is a `TableRegressionModel`, so you could just do `regmodel.model.rr.mu` and remove `_rr`.